lib/eddystone-beacon-scanner.js

```diff
@@ -38,6 +38,10 @@ EddystoneBeaconScanner.prototype.startScanning = function(allowDuplicates) {
 
   startScanningOnPowerOn();
 
+  if (gracePeriod) {
+    EXIT_GRACE_PERIOD = gracePeriod;
+  }
+
```

Comment:
I'd suggest we treat `EXIT_GRACE_PERIOD` like a constant instead, so something like this:

```javascript
this._gracePeriod =  (gracePeriod === undefined) ? EXIT_GRACE_PERIOD : gracePeriod;  
```

then replace all `EXIT_GRACE_PERIOD` references with `this._gracePeriod`. Maybe rename `EXIT_GRACE_PERIOD` to `DEFAULT_EXIT_GRACE_PERIOD` as well.

What do you think?